### PR TITLE
Update/react native 0.59.10

### PR DESCRIPTION
**Pull request by Thomazella**
💯 Updates react-native to 0.59.10.
Screen Shot 2019-08-25 at 5 08 34 PM

💅 Recreates android icons using android studio to fix white border.
Screen Shot 2019-08-25 at 4 37 07 PM
Screen Shot 2019-08-25 at 5 08 10 PM

🐛 Temporarily comment out some permission checks in MainActivity.java because it was crashing my app after upgrading. It seems to receive a -1 and call itself again. Not sure how to fix, help wanted.
Screen Shot 2019-08-25 at 5 36 36 PM

I've been pretty busy but I'm glad to send a lil PR. :D

EDIT: forgot to run tests after upgrading. I'll do that.

### CODE REVIEW

File: android/app/src/main/java/com/buttercup/MainActivity.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ public void checkPerms() {
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
         if (requestCode == OVERLAY_PERMISSION_REQ_CODE) {
-            checkPerms();
+            // checkPerms();
```
Comment (Thomazella) [inline android/app/src/main/java/com/buttercup/MainActivity.java]: this.

Comment (perry-mitchell) [reply android/app/src/main/java/com/buttercup/MainActivity.java]: Thanks @Thomazella - I can definitely take to testing that on Android :)

**Comment by perry-mitchell** [comment]
> forgot to run tests after upgrading. I'll do that.

They're failing most of the time now, so that's a separate issue. For now we're happy with a full hands-on test, which we'll help perform. We always check each connection type, loading and saving, before we release anyhow.

**Comment by Thomazella** [comment]
OK great. I'll do some testing shortly then

**Comment by Thomazella** [comment]
@perry-mitchell asked some android devs for advice and they linked me this doc https://developer.android.com/training/permissions/requesting I think they suggest to ask for the overlay permission in the manifest. I'm not entirely sure if we can/should do that in buttercup but just wanted to share this info :)

**Comment by perry-mitchell** [comment]
Hi @Thomazella,

> they suggest to ask for the overlay permission in the manifest

I think we already do that though:

https://github.com/buttercup/buttercup-mobile/blob/master/android/app/src/main/AndroidManifest.xml#L4-L6

**Comment by Thomazella** [comment]
Ah ok. :) great job on Google drive btw.

**Comment by perry-mitchell** [comment]
I'll close this in favour of #182.

I'm terribly sorry we didn't get to yours first - the other upgrade was an accident, as we were simply trying to fix android. We ended up going back and forth between rn 0.60 and 0.59, and just settled on this. We also rebuilt the x64 binaries as we found some substantial issues there.

We appreciate the effort you went to on this!

**Comment by Thomazella** [comment]
No big deal. :)

**Pull request closed without merging**